README.md

```diff
@@ -250,6 +250,19 @@ Please check the [CMake build instructions](cmake/README.md).
   cmake --build build --config Release --target install -v -- DESTDIR=install
   ```
 
+- Installing cpu_features(vcpkg)
+
+  You can download and install cpu_features using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
+  ```sh
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    ./vcpkg install cpu_features
```

Comment:
invalid command:
![image](https://user-images.githubusercontent.com/43444946/187976207-20b5b810-2c54-49c7-9cb5-be653ebb71e1.png)
could you replase `cpu_features` to `cpu-features`?

Reply:
Done, my bad, thanks for your review.